frontend/src/employee-mobile-frontend/api/attendances.ts

```diff
@@ -57,6 +58,29 @@ interface Absence {
   id: string
 }
 
+export type TimeRange = {
+  start: string
+  end: string
+}
+
+type DailyServiceTimesRegular = {
+  regular: true
+  regularTimes: TimeRange
+}
+
+type DailyServiceTimesIrregular = {
+  regular: false
+  monday: TimeRange | null
+  tuesday: TimeRange | null
+  wednesday: TimeRange | null
+  thursday: TimeRange | null
+  friday: TimeRange | null
+}
+
+export type DailyServiceTimes =
+  | DailyServiceTimesRegular
+  | DailyServiceTimesIrregular
```

Comment:
Recently we've started using `lib-common` for typings that are shared between multiple frontends. These could potentially also be extracted there instead of duplicating, although since these don't exactly use the same endpoint one could argue that the typings are also separate and just happen to be identical. So just FYI.

Reply:
https://github.com/espoon-voltti/evaka/tree/master/frontend/src/lib-common/api-types

Reply:
Moved to somewehere under `lib-common/api-types`, but not sure about file naming.